### DISCUSSION TITLE
How to expand the children of the book note over 2 level in the preview panel

### DISCUSSION THREAD

**Discussion by YQGong** [TriliumNext/Trilium#6331] [Q&A] [answered]
Expand all Children button only expand the first level content and the second level title. How to expand more than that?

![image](https://user-images.githubusercontent.com/32567917/118610633-8e27bb00-b7ee-11eb-9315-08632e7fd27e.png)

**Comment by zadam** [comment]
Hi, click on this button to expand more:

![image](https://user-images.githubusercontent.com/617641/118702890-c8588300-b815-11eb-920f-a15c4b7fb3c0.png)

**Comment by YQGong** [comment]
Is there a button to expand every possible children at next level or expand every children?

**Reply by eliandoran** [reply, marked as answer]
There is an expand button now in the ribbon, in the "Book" / "Collection" tab.
